src/test/java/edu/harvard/hul/ois/fits/junit/FitsOutputTest.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright 2009 Harvard University Library
+ *
+ * This file is part of FITS (File Information Tool Set).
+ *
+ * FITS is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * FITS is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with FITS.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package edu.harvard.hul.ois.fits.junit;
+
+import edu.harvard.hul.ois.fits.FitsMetadataElement;
+import edu.harvard.hul.ois.fits.FitsOutput;
+import edu.harvard.hul.ois.fits.tests.AbstractLoggingTest;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FitsOutputTest extends AbstractLoggingTest {
+
+    @Test
+    public void testFitsOutput() throws Exception {
```

Comment:
This test appears to pass with or without the changes you made to FitsOutput.

Reply:
Not for me.

Reply:
Following current Docker build & running tests:
```
# The build only needs to be run once
docker build -f docker/Dockerfile-test -t fits-test .
docker run --rm -v `pwd`:/fits -v ~/.m2:/root/.m2 fits-test mvn clean test
```
Some errors on running tests which appear unrelated to my changes:
```
Running edu.harvard.hul.ois.fits.consolidation.OISConsolidatorTest
[TEST] 2022-12-15 09:03:00 - ERROR - ToolBelt:144 - Thread [1] Error instantiating class: edu.harvard.hul.ois.fits.tools.jhove.Jhove -- Exception thrown: java.lang.reflect.InvocationTargetException -- Error message: null
[TEST] 2022-12-15 09:03:18 - ERROR - ToolBelt:144 - Thread [1] Error instantiating class: edu.harvard.hul.ois.fits.tools.droid.Droid -- Exception thrown: java.lang.reflect.InvocationTargetException -- Error message: null
[TEST] 2022-12-15 09:03:18 - ERROR - Fits:683 - Tool error processing file: image-vectorgraphic.svg
null
java.lang.reflect.InvocationTargetException: null
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
	at edu.harvard.hul.ois.fits.tools.ToolBelt.createToolClassInstance(ToolBelt.java:174) ~[classes/:?]
	at edu.harvard.hul.ois.fits.tools.ToolBelt.init(ToolBelt.java:127) ~[classes/:?]
	at edu.harvard.hul.ois.fits.tools.ToolBelt.<init>(ToolBelt.java:56) ~[classes/:?]
	at edu.harvard.hul.ois.fits.Fits.<init>(Fits.java:230) ~[classes/:?]
	at edu.harvard.hul.ois.fits.consolidation.OISConsolidatorTest$3.<init>(OISConsolidatorTest.java:153) ~[test-classes/:?]
	at edu.harvard.hul.ois.fits.consolidation.OISConsolidatorTest.jhoveBeforeDroid(OISConsolidatorTest.java:153) ~[test-classes/:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
(elided)
Caused by: java.lang.ClassCastException: class org.apache.commons.configuration.SubnodeConfiguration cannot be cast to class org.apache.commons.configuration.HierarchicalConfiguration (org.apache.commons.configuration.SubnodeConfiguration is in unnamed module of loader 'app'; org.apache.commons.configuration.HierarchicalConfiguration is in unnamed module of loader edu.harvard.hul.ois.fits.tools.utils.ParentLastClassLoader$ChildClassLoader @2c5529ab)
	at edu.harvard.hul.ois.fits.tools.utils.XsltTransformMap.getMap(XsltTransformMap.java:35) ~[classes/:?]
	at edu.harvard.hul.ois.fits.tools.jhove.Jhove.<init>(Jhove.java:77) ~[classes/:?]
	... 43 more
```

Reply:
The reason for this jdom2 fix is that a similar test in drs2_services was also failing.

Reply:
@daveneiman Interesting. That appears to be a class loader issue. Can you please try changing line 60 in `ParentLastClassLoader` to the following:

```java
loadByParentClassLoader.add("org.apache.commons.configuration");
```

Reply:
> The reason for this jdom2 fix is that a similar test in drs2_services was also failing.

I don't doubt that there may be an issue here, I would just like to have a reproducible failing test for it.

Reply:
@pwinckles I modified `ParentLastClassLoader` as request but same errors.

Reply:
Weird. Can you try turning on the trace logging for that class? It's unclear to me how `org.apache.commons.configuration.HierarchicalConfiguration` could be being loaded on the child classloader after making this change. It's also weird that your system is behaving so unusually, especially considering the tests are being executed from a docker container. Can you try completely rebuilding your test image?

```
docker build --no-cache -f docker/Dockerfile-test -t fits-test .
```

Reply:
I've run the tests (up until the first couple of errors) with 'trace' on for `ParentLastClassLoader`.
I have a 2GB file for you. How best to send it your way?

Reply:
If you zip it, how big is it?

Reply:
I meant 2.2MB (not GB). Zips down to 42K.

Reply:
Ah! That's a big difference! :)

If you aren't comfortable attaching inline here, you could send it to me on slack.